schema/PharmGKBIDs.mcf

```diff
@@ -0,0 +1,176 @@
+# generated by script based on dictionaries from
+# /scripts/pharmgkb/drug_gene_relations/config.py , then modified by hand
+
+Node: dcid:alfredID
+name: "alfredID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:comparativeToxicogenomicsDBID
+name: "comparativeToxicogenomicsDBID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:genAtlasID
+name: "genAtlasID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:geneCardID
+name: "geneCardID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:geneOntologyID
+name: "geneOntologyID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:humanCycGeneID
+name: "humanCycGeneID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:modBaseID
+name: "modBaseID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:mutDBID
+name: "mutDBID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:ucscGenomeBroswerID
+name: "ucscGenomeBroswerID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:iupharReceptorID
+name: "iupharReceptorID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:chebiID
+name: "chebiID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Drug,dcs:ChemicalCompound
```

Comment:
@tjann, since a Drug is a subClass of ChemicalCompound should we just list the domain as ChemicalCompound or should we list both?

Reply:
Yeah, we can just list ChemicalCompound